src/screens/Main/EditStack/UploadingVideoScreen/styles.tsx

```diff
@@ -0,0 +1,82 @@
+import { StyleSheet } from 'react-native'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#fff',
+    color: '#000',
+  },
+  title: {
+    fontSize: 40,
+  },
+  main: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  ellipses: {
+    color: '#E5186E',
+    fontSize: 200,
+    letterSpacing: -50,
+  },
+  text: {
+    fontFamily: 'montserrat-regular', 
+    fontStyle: 'normal',
+    fontWeight: 'bold',
+    fontSize: 30,
+    lineHeight: 37,
+    textAlign: 'center',
+    color: '#E5186E',
+    height: 114,
+    width: 307,
+    marginBottom: '45%',
+  },
+  submittedText: {
+    padding: 20,
+    marginBottom: '0%',
+    fontFamily: 'montserrat-bold', 
+    fontStyle: 'normal',
+    fontWeight: 'bold',
+    fontSize: 30,
+    lineHeight: 37,
+    textAlign: 'center',
+    color: '#E5186E',
+    height: 114,
+    width: 307,
+  },
+  subtext: {
+    alignContent: 'center',
+    fontFamily: 'roboto-regular',
+    fontSize: 16,
+    color: '#727272',
+    marginVertical: '5%',
+    textAlign: 'center',
+    width: 307,
+    marginTop: '0%',
+    marginBottom: '19%',
+  },
+  button: {
+    marginBottom: '15%',
+    width: 200,
+    height: 50,
```

Comment:
remove height and instead add vertical padding to buttonText. That way you can get rid of that janky marginBottom: -10% that is making the text not always be centered vertically